source/js/components/nsf-nav.js

```diff
@@ -32,9 +32,14 @@ var MenuLinks = React.createClass({
   },
   onScroll: function() {
     var links = this.state.links || [];
+    var hash = window.location.hash.replace(`#`, ``);
     var active = links[0] || ``;
     var scrollY = window.scrollY;
 
+    if (hash && !document.querySelector(`.nav-anchor#` + hash)) {
```

Comment:
omg i really need to remember to "submit review". Are you able to see my comment above now @ScottDowne ? https://github.com/mozilla/nsf-challenge/pull/25/files#r100611138